### PR TITLE
Feature/addsAromaMessages

### PR DESCRIPTION
**Pull request by cordhernandez**
BlackSourceLabs/BlackNectar-iOS#55

### CODE REVIEW

File: BlackNectar/BlackNectar/SideMenuFilterViewController.swift [hunk 1]
```diff
@@ -112,6 +114,10 @@ class SideMenuFilterViewController: UITableViewController {
     @IBAction func applyButton(_ sender: UIButton) {
         
         self.delegate?.didApplyFilters(self, restaurants: self.isRestaurant, stores: self.isStore, openNow: self.isOpenNow, distanceInMiles: self.distanceFilter)
+        AromaClient.beginMessage(withTitle: "Apply button selected")
```
Comment (SirWellington) [inline BlackNectar/BlackNectar/SideMenuFilterViewController.swift]: Titles are nice when they are Title Cased:
`Apply Button Selected`.

File: BlackNectar/BlackNectar/StoresMapViewController.swift [hunk 2]
```diff
@@ -36,6 +37,11 @@ class StoresMapViewController: UIViewController, MKMapViewDelegate, CLLocationMa
         super.viewDidLoad()
         
         prepareMapView()
+        AromaClient.beginMessage(withTitle: "User entered mapView screen")
+            .addBody("User is interacting with the map")
```
Comment (SirWellington) [inline BlackNectar/BlackNectar/StoresMapViewController.swift]: How do you feel about including the user's current location?

File: BlackNectar/BlackNectar/StoresTableViewController.swift [hunk 3]
```diff
@@ -73,6 +74,8 @@ class StoresTableViewController: UITableViewController, SideMenuFilterDelegate {
         if let revealController = self.revealViewController() {
             revealController.revealToggle(animated: true)
         }
+        
+        AromaClient.sendMediumPriorityMessage(withTitle: "Filter Opened")
```
Comment (SirWellington) [inline BlackNectar/BlackNectar/StoresTableViewController.swift]: This feels like a low-priority message to me.

File: BlackNectar/BlackNectar/StoresMapViewController.swift [hunk 4]
```diff
@@ -131,6 +137,11 @@ class StoresMapViewController: UIViewController, MKMapViewDelegate, CLLocationMa
             
         }
         
+        AromaClient.beginMessage(withTitle: "Getting directions")
```
Comment (SirWellington) [inline BlackNectar/BlackNectar/StoresMapViewController.swift]: Just so you know, you can also use a short-hand:
````swift
AromaClient.sendMediumPriorityMessage(withTitle: "Filter Opened", withBody: "User is getting directions to store: ")
```

File: BlackNectar/BlackNectar/StoresMapViewController.swift [hunk 5]
```diff
@@ -131,6 +137,11 @@ class StoresMapViewController: UIViewController, MKMapViewDelegate, CLLocationMa
...
+            .addBody("User is getting directions to location via Apple Maps")
```
Comment (SirWellington) [inline BlackNectar/BlackNectar/StoresMapViewController.swift]: It would be nice to include which store the user is getting location to.

**Review by SirWellington — APPROVED** [review]
Fresha than newborn.

File: BlackNectar/BlackNectar/SideMenuFilterViewController.swift (hunk 1 above)
Comment (cordhernandez) [reply BlackNectar/BlackNectar/SideMenuFilterViewController.swift]: got it! On the Job!

File: BlackNectar/BlackNectar/StoresMapViewController.swift (hunk 2 above)
Comment (cordhernandez) [reply BlackNectar/BlackNectar/StoresMapViewController.swift]: 👊🏾

File: BlackNectar/BlackNectar/StoresMapViewController.swift (hunk 5 above)
Comment (cordhernandez) [reply BlackNectar/BlackNectar/StoresMapViewController.swift]: On the Job!

File: BlackNectar/BlackNectar/StoresMapViewController.swift [hunk 6]
```diff
@@ -131,6 +137,8 @@ class StoresMapViewController: UIViewController, MKMapViewDelegate, CLLocationMa
             
         }
         
+        AromaClient.sendMediumPriorityMessage(withTitle: "Filter Opened", withBody: "User is getting directions to store: \(selectedPin)")
```
Comment (SirWellington) [inline BlackNectar/BlackNectar/StoresMapViewController.swift]: Wrong title. Should be, "Navigating to Store" or something like that.

**Pull request merged**